content/webapp/components/ExhibitionCaptions/ExhibitionCaptions.tsx

```diff
@@ -131,12 +131,12 @@ const Transcription = styled(Space).attrs({
 `;
 
 type Stop = {
-  standaloneTitle: prismicT.RichTextField;
+  standaloneTitle: string;
```

Comment:
If you look at how we're actually transforming `standaloneTitle`, we were calling `asText(…)`, which returns a `string`.

If you transform it as rich text instead, this component starts whinging because we're trying to use a raw object as a React element, i.e.

```html
<StandaloneTitle id={dasherize(`${standaloneTitle}`)}>
  {standaloneTitle}
</StandaloneTitle>
```